conflict_resolution.html

```diff
@@ -0,0 +1,196 @@
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whatever method of conflict resolution they feel comfortable with.</p>
+                
+            <p>Individuals are encouraged to try and resolve issues themselves in the first instance. However, if individuals do not feel comfortable or safe doing so, they can get help through the ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Individuals should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+
+        <section id="procedures">
+            <h2>Conflict Resolution Procedures</h2>
+
+            <p>When a conflict arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the violation, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in conflicts before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms, including reaching out to the person directly, reporting the issue to a chair or ombudsperson, or to a member of the team.</li>
```

Comment:
Reaching out to the person directly is a step that can be taken, but it is not a form of reporting.   It doesn't belong in the list of ways to report the incident.